### PR TITLE
Overall locale upgrade in what can be translated, Spanish Locale, French Locale (fully complete) and Image update.

### PR DESCRIPTION
**Pull request by Dormant-Hero**
Okay so ALH updated the Spanish locale to include translations. ALH noticed an incorrect image so I updated that as well.

Reward shop now includes code so that locales can be added for other languages. As a result I added English locale to all languages but Spanish for the reward shop items. Spanish locale working for reward shop items on my locale host.

Any other changes made are my editor removing unused imports. Hopefully not an issue.

### CODE REVIEW

File: src/locales/es-ES.ts [hunk 1]
```diff
@@ -80,19 +80,131 @@ export default {
   'app.currentpw': 'Contraseña actual',
   'app.newpw': 'Nueva Contraseña',
   'app.updateinfo': 'Actualizar Cuenta',
-  'app.minimum': 'Mí­nimo',
+  'app.minimum': 'Mí¬nimo',
```
Comment (ItsLaguna) [inline src/locales/es-ES.ts]: 'app.minimum': 'Mínimo',

Comment (Dormant-Hero) [reply src/locales/es-ES.ts]: Okay I will try to add to this

**Comment by Dormant-Hero** [comment]
Looks like I did not save somethign as well so the alh correction commit also removes my accident of leaving the number 11 in the slotData.head as part of testing how your code worked.

image

**Comment by Dormant-Hero** [comment]
Sorry also found that I left a console.log() for testing purposes within the code. Hopefully I have caught everything now.

**Comment by Dormant-Hero** [comment]
Only thing I have not touched or done much work for now is the FAQ page. A lot of stuff from error 404, login page, reward shop, and more now can be translated.

**Pull request merged**